source/geometry/IntVector2D.ooc

```diff
@@ -23,7 +23,7 @@ IntVector2D: cover {
 	x, y: Int
 	area ::= this x * this y
 	square ::= this x == this y
-	empty ::= !(this x > 0 && this y > 0)
+	isZero ::= this area <= 0
```

Comment:
No, don't add it here, we're gonna update the Float class to compare with tolerance.